index.html

```diff
@@ -320,9 +320,59 @@ <h2>Skills</h2>
                     </article>
                 </article>
             </article>
+            <!-- contact section -->
```

Comment:
- [OPTIONAL] This is not part of this milestone but I strangely suggest you stage this into consideration, Currently, the rest of your portfolio is either not visible or the structure is not all that good. You are building your portfolio section by section, but you should try to make sure that as you are working on the next milestone, the codes you have written before are still intact and the UI looks as it should. I'll suggest you check your CSS properly to make sure that the `header, works and language ` section still looks like the Figma design. :+1: :+1: :100: 

- - Video:

https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/107632780/8c00d0c2-2a4f-46bf-8ec8-40cd82102a53



- This is a placeholder text that suggests to you how to customize your readme me and it should be revised. I'll suggest you remove this to make your readme look more professional. :+1: :+1: 
- - Screenshot:
![Screenshot from 2023-06-14 10-10-39](https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/107632780/9c737779-d281-42dd-aa85-dc5a526c0056)

Reply:
Hello @V-Blaze I have made the changes you requested but concerning the optional suggestion on why my the other sections are not displaying I am wondering why that is the case since when I open the index.html file using live server it displays correctly. However, when I open it using githack the background I set on style.css are not displaying.
What could be the problem?